crates/common/tedge_users/src/unix.rs

```diff
@@ -33,6 +33,17 @@ struct InnerUserManager {
     guard: Option<users::switch::SwitchUserGuard>,
 }
 
+impl Default for UserManager {
+    fn default() -> Self {
```

Comment:
There is already a discussion about this [controversial use of `Default`](https://github.com/thin-edge/thin-edge.io/pull/8410)